### PR TITLE
[MIG] sale_wishlist: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**
Requires module:
- `sale_product_set` migrated in https://github.com/OCA/sale-workflow/pull/1399

### CODE REVIEW

**Review by petrus-v — APPROVED** [review]
Looks good to me by reading code source

**Review by rousseldenis — APPROVED** [review]

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo14-addon-sale-product-set @ git+https://github.com/trobz/sale-workflow@14.0-mig-sale_product_set#subdirectory=setup/sale_product_set
```
Comment (bguillot) [inline test-requirements.txt]: Hello @phuctranerp 

Can you remove this line since the dependency has been merged ? 

Thanks !

Comment (phuctranfxvn) [reply test-requirements.txt]: done thanks

**Pull request merged**